noarkexchange/src/main/java/no/difi/meldingsutveksling/noarkexchange/logging/PutMessageResponseMarkers.java

```diff
@@ -0,0 +1,21 @@
+package no.difi.meldingsutveksling.noarkexchange.logging;
+
+import net.logstash.logback.marker.LogstashMarker;
+import no.difi.meldingsutveksling.noarkexchange.schema.PutMessageResponseType;
+import no.difi.meldingsutveksling.noarkexchange.schema.StatusMessageType;
+
+import static no.difi.meldingsutveksling.logging.MarkerFactory.responseMessageCodeMarker;
+import static no.difi.meldingsutveksling.logging.MarkerFactory.responseMessageTextMarker;
+import static no.difi.meldingsutveksling.logging.MarkerFactory.responseTypeMarker;
+
+public class PutMessageResponseMarkers {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)